### PR TITLE
add verify success

### PR DESCRIPTION
**Pull request by icerove**
add tracking for success verification 2FA modal one

### CODE REVIEW

File: src/components/accounts/two_factor/TwoFactorVerifyModal.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ const TwoFactorVerifyModal = ({ open, onClose }) => {
     }
     
     const handleCancelClose = () => {
-        Mixpanel.track("2FA Modal Cancel verification")
```
Comment (vgrichina) [inline src/components/accounts/two_factor/TwoFactorVerifyModal.js:71]: not sure why this change, is `handleCancelClose` called in some other case than dismissing 2FA modal?

Comment (icerove) [reply src/components/accounts/two_factor/TwoFactorVerifyModal.js]: reverted

**Comment by icerove** [comment]
I look through 2FA verify modal again and find I misunderstanding `onClose` before, thanks for pointing out and this time it should be correct. @vgrichina
